Parsely/parsely-android
# Add unit tests setup

## Description

This PR adds unit tests setup for the SDK. I decided to introduce Robolectric to be able to test `AsyncTask` API.

It also adds support for measuring codecoverage via `kover` plugin, as well as integration with Codecov: https://app.codecov.io/github/Parsely/parsely-android

## How to test
CI should be enough: you could also run `koverHtmlReportRelease` task and see generated HTML report with 0% code coverage.
